largest_prime_factor.py

```diff
@@ -0,0 +1,27 @@
+"""
+The prime factors of 13195 are 5, 7, 13 and 29.
+
+What is the largest prime factor of the number 600851475143? 
+"""
+def is_prime(n):
+	if n <= 2:
+		return False
+	for x in range(2, num1):
+		if x ** 2 > n:
+			break
+		if n % x == 0:
+			return False
+	return True
+
+def factors(num1):
+	output = ()
+	if is_prime == False:
```

Comment:
you're checking if a function is false here?

Reply:
well I tried to first check if it was prime and if it was not a prime I wanted to keep dividing until a prime was met but i could not get passed this section.